topostats/default_config.yaml

```
@@ -38,9 +38,9 @@ grains:
     upper: 1.0
   direction: upper # Options: upper, lower, both (defines whether to look for grains above or below thresholds or both)
   # Thresholding by area
-  absolute_smallest_grain_size: 100
+  smallest_grain_size: 24 # Size in nm^2 of tiny grians/blobs (noise) to remove.
```

Might be useful to include the units in the variable name...

```suggestion
  smallest_grain_size_nm2: 24 # Size in nm^2 of tiny grians/blobs (noise) to remove.
```